vars/runATH.groovy

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env groovy
+
+/**
+ * Wrapper for running the ATH
+ */
+
+def call(Map params = [:]) {
+    def String athUrl = params.get('athUrl', 'https://github.com/jenkinsci/acceptance-test-harness.git')
```

Comment:
This is unnecessary on two levels. You never need both String and an object type and you don't need to call get on an element in a map in groovy. The preferred way to do it is via the groovy properties, http://groovy-lang.org/objectorientation.html has more details.

Reply:
`params.get(key, default)` is the recommended way of default in pipeline since [script-security#153](https://github.com/jenkinsci/script-security-plugin/pull/153)